kong/globalpatches.lua

```diff
@@ -68,6 +68,75 @@ return function(options)
   end
 
 
+  do
+    local timer_module = require("resty.timerng")
+    local timerng
+
+    if options.cli or options.rbusted then
+      timerng = timer_module.new({
+        min_threads = 16,
+        max_threads = 32,
+      })
+      timerng:start()
+
+    else
+      timerng = timer_module.new()
+    end
+
+    local native_timer_at = ngx.timer.at
+    local native_timer_every = ngx.timer.every
+
+    local is_patched = false
+
+    _G.ngx.timer.at = function (delay, callback, ...)
```

Comment:
I think `_G` is unnecessary.

Reply:
done

Reply:
This is purely a personal preference to highlight that it is they are global.

Reply:
I think the `_G` is fine, it's in line with the rest of that module.